app/views/content/train-to-be-a-teacher/school-placements.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Teacher training school placements"
+heading: "Teacher training school placements"
+subcategory: Postgraduate teacher training
+description: |-
+  Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available. 
+navigation: 20.20
+navigation_title: Teacher training school placements
+navigation_description: Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 
+quote:
+  q-giorgio:
+    text: "The highlight of my placements was getting to teach!"
+    name: "Giorgio, biology teacher"
+    classes: quote--indent
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+keywords:
+    - placements
+    - school placements
+    - classroom placements
+    
+---
+
+School placements are an important and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:
+
+* get experience in the classroom
+* immerse yourself in your placement schools
+* get to know your pupils and colleagues
+* start your career as a teacher
```

Comment:
Wonder if we could pull out the colleagues bit here, so become part of a school community and learn from other teachers, or something like that